### PR TITLE
click and drag

### PR DESCRIPTION
**Pull request by therealmbittarelli**
adding functionality to allow user to click and drag images

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -6,14 +6,36 @@
...
 <body>
-  <p>
-    <h3>hello world</h3>
-  </p>
+  <main>
+    <section id="gandalf">
+      <h3 id="gandalfheader">hello, gandalf!</h3>
+    </section>
+    <section class="img">
+      <div id="mydiv">
```
Comment (mbittarelli) [inline client/index.html]: please more descriptive name for div id

File: client/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "hello": "echo 'hello client'",
     "start": "http-server ./dist -p 3333",
-    "build": "cp ./index.html ./dist && webpack --config webpack.config.js"
+    "build": "cp ./index.html ./dist && cp ./main.css ./dist && mkdir -p ./dist/img && cp -R ./img/* ./dist/img && webpack --config webpack.config.js"
```
Comment (mbittarelli) [inline client/package.json:6]: please split up in multiple lines for readability

File: client/src/main.ts [hunk 3]
```diff
@@ -1 +1,139 @@
-console.log('hello world');
\ No newline at end of file
+console.log('hello world');
```
Comment (mbittarelli) [inline client/src/main.ts]: please remove

File: client/src/main.ts [hunk 4]
```diff
@@ -1 +1,139 @@
...
+
+// Make the DIV element draggable:
+dragElement(document.getElementById('mydiv'));
```
Comment (mbittarelli) [inline client/src/main.ts]: please use typescript types

**Review by mbittarelli — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 2 above)
Comment (therealmbittarelli) [reply client/package.json:6]: json doesn't support multi-line strings; unable to make this change

**Review by mbittarelli — APPROVED** [review]

**Pull request merged**